README.markdown

```diff
@@ -151,3 +151,8 @@ maintained by the RSpec team but may receive support and maintenance as needs re
 * [https://github.com/rspec/rspec-collection_matchers](https://github.com/rspec/rspec-collection_matchers)
 * [https://github.com/rspec/rspec-its](https://github.com/rspec/rspec-its)
 * [https://github.com/rspec/rspec-legacy_formatters](https://github.com/rspec/rspec-legacy_formatters)
+
+## Ruby Engine Support
+* MRI is officially supported by RSpec.
+* TruffleRuby is not officially supported by RSpec and any issues with this engine should be reported to [https://github.com/oracle/truffleruby/issues](https://github.com/oracle/truffleruby/issues).
```

Comment:
This almost feels like a step backward.
Maybe we should just go for passing all of RSpec CIs, add TruffleRuby in those CIs, and aim for official support?